files/dx_sync_directory.py

```diff
@@ -393,9 +393,9 @@ def split_into_tar_files(files_to_upload, log, args):
     tars_to_upload.append(current_tar)
 
     if total_size < args.min_tar_size:
-        print >> sys.stderr, ('QUITTING: Size of files to upload is not big ' +
+        print(('QUITTING: Size of files to upload is not big ' +
```

Comment:
why double brackets? (same comment as before, just close this comment if there's an explanation)

Reply:
yes. will clean that